MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDelegate.java

```diff
@@ -95,7 +95,7 @@ public float getElevation() {
      * @see View#setElevation(float)
      */
     public void setElevation(float elevation) {
-        boolean needsWrap = elevation > mElevation;
+        boolean needsWrap = elevation != mElevation;
```

Comment:
I'm not quite sure I understand what's happening 😅  Still, I trust your judgement so let's move on, it's a simple change 👍